docs/index.html

```diff
@@ -76,7 +76,6 @@
         rows[i].onclick = function () {
             var cssClass = this.getAttribute( 'class' );
             var iconID = cssClass.split(' gridicons-')[1];
-			alert(iconID);
```

Comment:
Good question, honestly I don't know. That file (and svg-sprite/index.html) are generated by a grunt task I believe.

I'll look into it later but probably safest to just remove.